Plugins/ReplaceTimestamps/ReplaceTimestamps.plugin.js

```diff
@@ -111,7 +112,7 @@ module.exports = !global.ZeresPluginLibrary
                 const date = new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '').replace(/\d?\d:\d\d/, time);
                 const then = Math.round((new Date(date)).getTime() / 1000);
                 if (isNaN(then)) return time;
-                return `<t:${then}:t>`;
+                return `<t:${then}:${this.format ? this.format : "t"}>`;
```

Comment:
Then you can do this here:
```suggestion
                return `<t:${then}:${format}>`;
```